husky_description/meshes/accessories/bumblebee2.dae

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why not use the mesh from the driver: https://github.com/ros-drivers/pointgrey_camera_driver/tree/master/pointgrey_camera_description